moveit_core/collision_detection/include/moveit/collision_detection/collision_robot.h

```diff
@@ -163,27 +163,81 @@ class CollisionRobot
                                    const AllowedCollisionMatrix& acm) const = 0;
 
   /** \brief The distance to self-collision given the robot is at state \e state. */
-  virtual double distanceSelf(const robot_state::RobotState& state) const = 0;
+  inline double distanceSelf(const robot_state::RobotState& state) const
```

Comment:
All these old functions (that you deprecated) are now implemented as thin inline wrappers of the actual worker function. Please verify correctness.

Reply:
I am not sure what has happened but on my pc these changes are not present.

Reply:
It looks like this was a change you made in the commit "keep old functions as conviencency wrappers". It is interesting that you were able to change code in my repository.

I will let you make this correction then I will apply the other requested fixes.

Reply:
I was done with my changes. You can proceed ;-)

Reply:
@rhaschke Was it your intention to move these to the base class?